test/test_torch.py

```diff
@@ -2701,12 +2699,7 @@ def test_qtensor_creation(self):
         val = 100
         numel = 10
         q = torch._empty_affine_quantized(numel, dtype=torch.qint8, scale=scale, zero_point=zero_point)
-        # for i in range(numel):
-        #     # wait for th_fill
-        #     q[i] = val
-        # r = q.dequantize()
-        # for i in range(numel):
-        #     self.assertEqual(r[i], (val - zero_point) * scale)
+        # TODO: check dequantized values?
```

Comment:
cc @jerryzh168 
I don't know what these comments were for.  Let me know if you think this TODO is inaccurate and I will update accordingly.